### DISCUSSION TITLE
When moving to EGL, do I need to do anything to "move" cloud saves?

### DISCUSSION THREAD

**Discussion by ghost** [legendary-gl/legendary#199] [Q&A] [answered]
If for some reason Legendary was disabled from working due to Epic changes on their backend, would I need to do anything to move/upload my cloud saves to work with EGS again, or would EGS detect saves and do what it needs on its own, to transition from Legendary back to EGS?

**Comment by CommandMC** [comment]
That depends on how EGL handles saves (legendary up-/downloads the saves the same way EGL does and places them into the same directorys). If Epic did an even remotely good job, EGL checks the Modify dates of the local file and the online one and asks which one to keep (same way Steam does it), but only way to know for sure is to install EGL

**Comment by derrod** [comment, marked as answer]
The Epic Launcher will ask you if you want to download a cloud save the first time you try to launch a supported game if a local save exists, otherwise it should just download the save normally.

![egl_screenshot](https://user-images.githubusercontent.com/3123295/104138690-16600480-53a6-11eb-9440-c8576843e654.png)
